Readme.md

```diff
@@ -37,22 +37,13 @@ If you want to debug golang backend plugin code, follow the steps below:
 * Copy this processId to the `.vscode/launch.json`
 * In your VSCode from 'Debug' menu call 'Start Debugging'
 
-
-
 ## Documentation
 
-> Please refer to  <https://docs.oracle.com>.
-
-## Examples
-
-> N/A
+Please refer to  <https://docs.oracle.com>.
```

Comment:
This doesn't address the comment at all. It's still way too vague. If you don't have specific documentation for this plugin or the service to which it is associated, remove the section completely.